tests/support/support.ts

```diff
@@ -0,0 +1,112 @@
+/* eslint-disable @typescript-eslint/naming-convention */
+import type { RadiclePeer } from "./peerManager";
+
+import { fileURLToPath } from "url";
+import { dirname } from "path";
+import * as Path from "node:path";
+import * as Fs from "node:fs/promises";
+
+import { gitOptions } from "@tests/support/fixtures";
+
+const filename = fileURLToPath(import.meta.url);
+const dir = dirname(filename);
+
+// Create a project using uphe rad CLI.
+export async function createProject(
+  peer: RadiclePeer,
+  name: string,
+  description = "",
+): Promise<{ rid: string }> {
+  const cwd = Path.join(peer.checkoutPath, name);
+  await Fs.mkdir(cwd, { recursive: true });
+  await peer.git(["init", "--initial-branch", "master"], {
+    cwd,
+    env: gitOptions["alice"],
+  });
+  await peer.git(["commit", "--allow-empty", "--message", "initial commit"], {
+    cwd,
+    env: gitOptions["alice"],
+  });
```

Comment:
As you can see here you manually have to pass `gitOptions` every time you interact with git for the peer. Instead you could pass the git options when you create the peer and then add the environment variables in `RadiclePeer#spawn()`.